be/src/exec/pipeline/olap_scan_operator.h

```diff
@@ -2,124 +2,12 @@
 
 #pragma once
 
-#include <optional>
+#include "exec/pipeline/pipeline_builder.h"
+#include "exec/pipeline/scan_operator.h"
+#include "exec/scan_node.h"
 
-#include "exec/pipeline/source_operator.h"
-#include "exec/workgroup/work_group_fwd.h"
-#include "exprs/vectorized/runtime_filter_bank.h"
-#include "runtime/global_dicts.h"
-#include "util/blocking_queue.hpp"
-#include "util/priority_thread_pool.hpp"
+namespace starrocks::pipeline {
 
-namespace starrocks {
+OpFactories decompose_olap_scan_node_to_pipeline(ScanNode* scan_node, PipelineBuilderContext* context);
```

Comment:
Is this method re-used by other derived ScanNode? if so, suggest to rename it to decompose_scan_node_to_pipelines, otherwise, keep OlapScanNode::decompose_to_pipelines unchanged.